charts/redpanda/chart_template_test.go

```diff
@@ -294,6 +296,171 @@ tls:
 	}
 }
 
+func CertTrustStoreCases(t *testing.T) []TemplateTestCase {
+	// truststores is a map of listener type to map of listener name to truststore_file. ({"admin": {"internal": "ca.crt"}}).
+	assertTrustStores := func(t *testing.T, manifests []byte, truststores map[string]map[string]string) {
+		cm, err := extractRedpandaConfigsFromConfigMap(manifests)
```

Comment:
`extractRedpandaConfigsFromConfigMap` is missing from this PR.

Reply:
Fixed!